workspaces/copilot/plugins/copilot-backend/README.md

```diff
@@ -90,14 +91,17 @@ copilot:
       seconds: 15
   host: YOUR_GITHUB_HOST_HERE
   enterprise: YOUR_ENTERPRISE_NAME_HERE
+  organization: YOUR_ORGANIZATION_NAME_HERE
```

Comment:
There will be many users of backstage that have multiple GH organisations. Could this be handled here?

Reply:
It's something I plan to implement later. For now, I'll be implementing only a simple organization, which will require some structural changes and adjustments to how things currently work.

Reply:
It's something I plan to implement later. For now, I'll be implementing only a simple organization, which will require some structural changes and adjustments to how things currently work.